packages/slice-machine/src/legacy/components/LoginModal/index.tsx

```diff
@@ -76,6 +78,10 @@ const LoginModal: React.FunctionComponent = () => {
         60,
       );
 
+      // refresh queries to update the UI
```

Comment:
❓ Does this also handle the refresh of the getState as you did during the logout?

Reply:
No that is already happening below.

Reply:
Hum where? Sorry I don't get it